ajax/libs/jquery-nearest/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "jquery-nearest",
+    "description": "Find the elements in a page that are closest to a particular point or element, based on pixel dimensions",
+    "filename": "jquery.nearest.min.js",
+    "keywords": [
+        "distance",
+        "ecosystem:jquery", 
+        "furthest", 
+        "jquery-plugin",         
+        "nearest",
+        "touching", 
+        "pixel"        
+    ],
+    "author": {
+        "name": "Gilmore Davidson",
+        "url": "https://shoehornwithteeth.com/"
+     },
+    "homepage": "https://gilmoreorless.github.io/jquery-nearest/",
+    "repository": {
+        "type": "git",      
+        "url": "git+https://github.com/gilmoreorless/jquery-nearest.git"
+    },    
+    "autoupdate": {
+        "source": "git",
+        "target": "git://github.com/gilmoreorless/jquery-nearest.git",
+        "fileMap": [
+          {
+            "basePath": "src",
+            "files": [
+              "jquery.nearest*.+(css|map|js)"
+            ]
+          }
```

Comment:
We can add another element of fileMap here to grab all versions of the lib. You can see the example in https://github.com/cdnjs/cdnjs/pull/11439. Let's add this:
```
{
  "basePath": "src",
  "files": [
    "jquery.nearest*"
  ]
}
```
Don't forget the comma between the elements :)
If would be good to do rebase and squash like what you tried to do in https://github.com/cdnjs/cdnjs/pull/11685. But don't worry I can still help you do that. Thank you.